src/Cheddar.js

```diff
@@ -1,3 +1,13 @@
 export { default as Controller } from './Controller'
 export { default as Model } from './Model'
 export { default as Router } from './Router'
+
+export class Cheddar {
+  static get database() {
+    return this._database
+  }
+
+  static set database(url) {
+    this._database = url
+  }
+}
```

Comment:
I won't add a test for a simple getter and setter. If it gets more complex we can add tests later.